WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -162,7 +183,7 @@ private async Task<bool> InitialiseTransactionAsync()
 				}
 			}
 
-			if (_info.FeeRate == FeeRate.Zero)
+			if (!_info.IsCustomFeeUsed && _info.FeeRate == FeeRate.Zero)
```

Comment:
This is not necessary because if `IsCustomFeeUsed` is true then `_info.FeeRate == FeeRate.Zero` is always false:

```suggestion
			if (_info.FeeRate == FeeRate.Zero)
```